src/lib/get-auth.ts

```diff
@@ -44,11 +29,24 @@ function getKey(user: User, scheme: SecurityScheme): authKey {
   }
 }
 
-// For `scheme` we're typing it to a union of `SecurityScheme` and `any` because we have handling and tests for an
-// unknown or unrecognized `type` and though it's not possible with the `SecurityScheme.type` to be unrecognized it may
-// still be possible to get an unrecognized scheme with this method in the wild as we have API definitions in our
-// database that were ingested before we had good validation in place.
-function getByScheme(user: User, scheme = <SecurityScheme | any>{}, selectedApp?: selectedAppType): authKey {
+/**
+ * Retrieve auth keys for a specific security scheme for a given user for a specific "app" that they have configured.
+ *
+ * For `scheme` we're typing it to a union of `SecurityScheme` and `any` because we have handling and tests for an
+ * unknown or unrecognized `type` and though it's not possible with the `SecurityScheme.type` to be unrecognized it may
+ * still be possible to get an unrecognized scheme with this method in the wild as we have API definitions in our
+ * database that were ingested before we had good validation in place.
+ *
+ * @param user User
+ * @param scheme Security scheme to get auth keys for.
+ * @param selectedApp The user app to retrieve an auth key for.
+ * @returns The found auth key for this security scheme.
+ */
+export function getByScheme(
+  user: RMOAS.User,
+  scheme = <RMOAS.KeyedSecuritySchemeObject | any>{},
```

Comment:
Added that so that these tests could still function where we're testing a case where an unknown schema type is present:

```
expect(getByScheme(topLevelUser, { type: 'unknown' })).toBeNull();
expect(getByScheme(keysUser, { type: 'unknown' })).toBeNull();
expect(getByScheme(keysUser, { type: 'unknown' }, 'app-2')).toBeNull();
```

Without `any` there:

> Type '"unknown"' is not assignable to type '"oauth2" | "http" | "apiKey" | "openIdConnect"'.ts(2322)
> index.d.ts(415, 9): The expected type comes from property 'type' which is declared here on type 'KeyedSecuritySchemeObject'

There a better way to handle this without `any`, or are these tests just not really necessary anymore?